src/RestWrite.js

```diff
@@ -1104,6 +1102,29 @@ RestWrite.prototype.sanitizedData = function() {
   return Parse._decode(undefined, data);
 }
 
+// Returns an updated copy of the object
+RestWrite.prototype.buildUpdatedObject = function (extraData) {
+  var updatedObject = triggers.inflate(extraData, this.originalData);
```

Comment:
PS here it's in fact a `const` variable , so I've replaced `var` with `const` anyway, but the style question still stands.